testutil/localstride/scripts/setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
```

Comment:
very nit, but should this file be named `start.sh` instead of `setup.sh` since it does more than just initialize state?